Source/CalendarStyle.swift

```diff
@@ -42,10 +42,23 @@ public class TimelineStyle {
   public var timeColor = UIColor.lightGray
   public var lineColor = UIColor.lightGray
   public var backgroundColor = UIColor.white
+  public var timeFormat = TimelineTimeFormat.system
+  public var density = CalendarDensity.regular.rawValue
   public init() {}
 }
 
 public class CurrentTimeIndicatorStyle {
   public var color = UIColor.red
   public init() {}
 }
+
+public enum TimelineTimeFormat {
+  case twentyFourHour, twelveHour, system
```

Comment:
I was going to decouple `CalendarConfig` to a separate class, similar to `CalendarStyle`. The concept is the same, the only difference that `CalendarStyle` is responsible for defining looks only, and `CalendarConfig` is about time/date/calendar settings in general.